package.json

```diff
@@ -124,7 +124,6 @@
     "platformicons": "^5.4.0",
     "po-catalog-loader": "2.0.0",
     "prettier": "2.7.1",
-    "prism-sentry": "^1.0.2",
```

Comment:
The new colors are based on `prism-sentry`, with slight modifications to increase contrast and readability. Removing the package because it is no longer needed (it's still being used in Sentry Docs).

Reply:
Are we okay fragmenting this?

Reply:
Yepp! Sentry Docs has a different color system than the Sentry app — it's much more playful and works well with `prism-sentry` (in fact, I believe `prism-sentry` was originally designed with Sentry Docs in mind).